debian/control

```diff
@@ -654,6 +669,18 @@ Description: Python libraries for the Ceph librbd library
  This package contains Python libraries for interacting with Ceph's
  RBD block device library.
 
+Package: python3-rbd
+Architecture: linux-any
+Section: python
+Depends: librbd1 (>= ${binary:Version}), ${misc:Depends}, ${python3:Depends}
+Description: Python libraries for the Ceph librbd library
+ Ceph is a massively scalable, open-source, distributed
+ storage system that runs on commodity hardware and delivers object,
+ block and file system storage.
+ .
+ This package contains Python libraries for interacting with Ceph's
```

Comment:
and i'd also recommend change the description of `python-rbd` to note that it contains the Python 2 library for interacting with Ceph's RBD block device library.

the same applies to other python libraries.